src/table/components/TableHeadWrapper.jsx

```diff
@@ -33,19 +34,9 @@ const useStyles = makeStyles({
   },
 });
 
-function TableHeadWrapper({
-  rootElement,
-  tableData,
-  theme,
-  layout,
-  changeSortOrder,
-  constraints,
-  translator,
-  selectionsAPI,
-  focusedCellCoord,
-  setFocusedCellCoord,
-  keyboard,
-}) {
+function TableHeadWrapper({ focusedCellCoord, setFocusedCellCoord }) {
```

Comment:
if we have our context as a separate layer, we can move the focused cell logic into the context by it's own custom hook and get all required stuff again from context

this is the core benefit of context fo us.